crypto/rand/rand_unix.c

```diff
@@ -217,7 +217,8 @@ static size_t sysctl_random(char *buf, size_t buflen)
      * Just return an error on older NetBSD versions.
      */
 #if   defined(__NetBSD__) && __NetBSD_Version__ < 400000000
-    return 0;
+    errno = ENOSYS;
+    return -1;
```

Comment:
There is another `return 0` few lines up.

Reply:
done, see b81dc2738bd575543720019ec5ad017517968098

Reply:
it's now in 7722d3d063098af9362eb486468bf3268887b07d.